lib/pascal.rb

```diff
@@ -1,36 +1,33 @@
+class Pascal_triangle
+  def draw(height=1)
+    spaces = " "
+    num_spaces = height
+    for row in (1..height)
+      puts "#{spaces*num_spaces}#{nth_row(row)}"
```

Comment:
You can move this line into the separate method e.g. `print_row(num_spaces, row)`.  That will separate responsibilities to different methods.